.github/workflows/build-deploy-stage.yml

```diff
@@ -0,0 +1,94 @@
+name: Deploy to Amazon ECS stage environment
+
+on:
+  push:
+    branches: [ "dev-branch-1", "main", "add-github-actions-cicd-pipeline" ]
+
+env:
+  AWS_REGION: us-west-2
+  ECR_REPOSITORY: civictechjobs-fullstack-stage
+  ECS_SERVICE: civictechjobs-fullstack-stage
+  ECS_CLUSTER: incubator-prod
+  ECS_TASK_DEFINITION: ./aws/task-definition.json
+  ECS_FAMILY: civictechjobs-fullstack-stage
+  ECS_TASK_ROLE_ARN: incubator-prod-ecs-task-role
+  ECS_EXECUTION_ROLE_ARN: incubator-prod-ecs-task-role
+
+  DB_HOST: 34.132.106.11
+  DB_NAME: my-test-db
```

Comment:
Are the credentials here yet? If not, we should merge this once the credentials are set.

Reply:
updated to point to incubator DB